pkgs/by-name/c2/c2FmZQ/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
+
+buildGoModule rec {
+  pname = "c2FmZQ";
+  version = "0.4.7";
+
+  src = fetchFromGitHub {
+    owner = "c2FmZQ";
+    repo = "c2FmZQ";
+    rev = "v${version}";
+    hash = "sha256-kJRzIpYa25BoylArppXW/ioznS7hwmnynh0rO+LOg7w=";
+  };
+
```

Maybe a few `ldflags`?

https://github.com/c2FmZQ/c2FmZQ/blob/1bf878c2260975a5c54d76b41c0e366a7c8c3aa0/.github/workflows/release.yml#L62-L71

```suggestion

ldflags = [ "-s -w" ];

```